src/binder/assemblybinder.cpp

```diff
@@ -1287,7 +1287,7 @@ namespace BINDER_SPACE
     }
     
     /*
-     * BindByTpaList is the entry-point for the custom binding algorithm on the Phone.
```

Comment:
...on the Phone? Is this just someone writing the comment while receiving a call on the phone, or was this code somehow related to Windows Phone initially?

Reply:
Funnily enough lot of the binder code we're using today comes from the Windows Phone code base   😄 .
I didn't touch this comment - there are many places where the comments are off in certain ways.. it's just that I don't feel comfortable yet to fix them up as I don't have full understanding of the area.